app/controllers/api/sequences_controller.rb

```diff
@@ -3,5 +3,15 @@ class SequencesController < Api::AbstractController
     def create
       mutate Sequences::Create.run(params, user: current_user)
     end
+
+    def destroy
+      # TODO: If you touch this again, add a mutation.
+      seq = Sequence.find(params[:id])
+      if (seq.user == current_user) && seq.destroy
+        render nothing: true
+      else
+        raise Errors::Forbidden, "Not your Sequence object."
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.